src/components/Success.jsx

```diff
@@ -0,0 +1,54 @@
+import { Link } from 'react-router-dom';
+
+const Success = () => {
+  return (
+    <>
+      <div className="h-screen flex flex-row">
+        <div className="h-full w-full lg:w-[450px] flex flex-col justifyevenly">
```

Comment:
Typo here justify-evenly not justifyevenly

Reply:
I forgot to delete it.